org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/settings/XMLValidationSettings.java

```diff
@@ -16,7 +16,8 @@
 import org.eclipse.lsp4j.PublishDiagnosticsCapabilities;
 
 /**
- * XMLValidationSettings
+ * XML validation settings for a given file.
```

Comment:
Is this class ever instantiated directly? If not, I think it might be worthwhile to rename it so that we don't confuse it with `XMLValidationRootSettings`. Maybe `XMLValidationFileSettings`?

Reply:
I would like to avoid doing that because it will break some APIs.

Reply:
Okay. I think we will get confused by the name in the future at some point, but if we really can't change it, then that's okay.